### ISSUE TITLE
Search is not working

### ISSUE THREAD

**Issue by ngeorgop** [ViDA-NYU/ARIES-Issues#16]
When I attempt to search/filter my images (say, by an artist's name), nothing happens. No change occurs within the image menu.

**Comment by joaorulff** [comment]
Hello

Thanks for your feedback.

Did you try to fill the corresponding (author name, for example) metadata fields for this specific project?

**Comment by ngeorgop** [comment]
Yes - the problem seems to have resolved itself when I logged out and logged back in. Thank you!
